FirstProject/FirstNumber.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+namespace FirstProject
+{
+    internal class FirstNumber
+    { 
+        public void PrimeNumber(int number3)
+        {
+            bool firstnumber = true;
```

Comment:
używajmy CamelCasea czyli jak już chcesz tak nazwać to firstNumber, ale prawidłowo powinno być to nazwane isPrime(prime bo prime number = liczba pierwsza)